lib/current_order.rb

```diff
@@ -0,0 +1,32 @@
+class CurrentOrder
+
+  attr_reader :order
+
+  def initialize
+    @order = []
+  end
+
+  def add_item(item)
+    @order << item
+  end
+
+  def total
+    final_price = 0
+    @order.each do |n| 
+      final_price += n[:price]
+    end
+    show_order
+    final_price
+  end
+
+  def clear_order
+    @order = []
+  end
+
+  private
+
+  def show_order
```

Comment:
Good call to use a separate method for this.  The next step would be to move it into a new class and leave your `CurrentOrder` to focus on the business logic of orders.